Brocfile.js

```diff
@@ -52,37 +52,51 @@ var localeDataFiles = config.availableLocales.map(function (locale) {
     return locale.split('-')[0] + '.js';
 });
 
-// Create a roll-up file of locale data for the locales the app supports for
-// each integration lib.
-var formatjsLocaleData = mergeTrees([
-    'dust-intl',
-    'handlebars-intl',
-    'react-intl'
-].map(function (integration) {
+function localeTree (srcDir, outputFile) {
     return concatTree(new Funnel(node_modules, {
-        srcDir : integration + '/dist/locale-data',
+        srcDir : srcDir,
         destDir: '/',
         files  : localeDataFiles
     }), {
         inputFiles: ['*.js'],
-        outputFile: '/vendor/formatjs/' + integration + '-locale-data.js'
+        outputFile: outputFile
     });
+}
+
+// Create list of locale data filenames for all of the locales the app supports,
+// which is a small subset of all the locales Format.js libs support.
+var localeDataFiles = config.availableLocales.map(function (locale) {
+    return locale.split('-')[0] + '.js';
+});
+
+var formatjsLocaleData = mergeTrees([
+    'dust-intl',
+    'handlebars-intl',
+    'react-intl'
+].map(function (integration) {
+    return localeTree(integration + '/dist/locale-data', '/vendor/formatjs/' + integration + '-locale-data.js');
 }));
 
+var emberIntlLocaleData = localeTree('ember-intl/packaging/dist/locale-data/locales', '/vendor/formatjs/ember-intl-locale-data.js');
```

Comment:
What else is in `ember-intl/packaging/dist/locale-data/` besides the `locales/` dir?